packages/turbo-ignore/package.json

```diff
@@ -2,18 +2,39 @@
   "name": "turbo-ignore",
   "version": "0.0.39",
   "description": "",
-  "main": "index.js",
-  "bin": "index.js",
+  "homepage": "https://turborepo.org",
   "keywords": [],
   "author": "Jared Palmer",
   "license": "MPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/vercel/turborepo",
+    "directory": "turbo-ignore"
+  },
+  "bugs": {
+    "url": "https://github.com/vercel/turborepo/issues"
+  },
+  "main": "index.js",
+  "bin": "index.js",
   "scripts": {
-    "build": "tsup index.js --format=cjs && cp ./package.json ./dist/package.json",
-    "release": "pnpm build && cd dist && npm publish"
+    "build": "tsup src/index.ts --format=cjs && cp ./package.json ./dist/package.json",
+    "release": "pnpm build && cd dist && pnpm publish",
+    "test": "jest",
+    "lint": "eslint src/**/*.ts"
   },
   "devDependencies": {
     "@manypkg/find-root": "^1.1.0",
+    "@types/fs-extra": "^9.0.13",
+    "@types/jest": "^27.4.0",
     "esbuild": "^0.14.38",
-    "tsup": "^5.12.1"
-  }
+    "execa": "5.1.1",
+    "fs-extra": "^10.1.0",
+    "jest": "^27.4.3",
+    "ts-jest": "^27.1.1",
+    "tsup": "^5.12.1",
+    "typescript": "^4.7.4"
+  },
+  "files": [
+    "dist"
+  ]
```

Comment:
So, funny story, since we're publishing from inside of dist *after moving `package.json`* I believe that this is wrong.

Reply:
yes this is wrong.

Reply:
You are correct, this is a remnant of moving this to package similarly to create-turbo